src/slic3r/GUI/CalibrationFlowDialog.cpp

```diff
@@ -82,23 +82,28 @@ void CalibrationFlowDialog::create_geometry(float start, float delta) {
     }
 
     //add sub-part after scale
-    float zshift = (1 - zscale) / 2;
     float zscale_number = (first_layer_height + layer_height) / 0.4;
+    /* zshift is calculated using the following:
+    (zscale / 2) represents the midpoint of the filament_flow_test_cube
+    ((first_layer_height + layer_height) / 2) represents the midpoint of our indicator tab (it is scaled to be 2 layers tall)
+    The 0.3 constant is the same as the delta calculated in add_part below, this should probably be calculated per the model object
```

Comment:
I did not want to modify the code used in add_part, but this can possible be better handled by creating a parameter for add_part that disables the delta calculation.